src/torch/test_cases.py

```diff
@@ -0,0 +1,164 @@
+import os
+import numpy
+# Import the PET reconstruction engine
+import sirf.STIR as pet
+# Set the verbosity
+pet.set_verbosity(1)
+# Store temporary sinograms in RAM
+pet.AcquisitionData.set_storage_scheme("memory")
+import sirf
+msg = sirf.STIR.MessageRedirector(info=None, warn=None, errr=None)
+from sirf.Utilities import examples_data_path
+import matplotlib.pyplot as plt
+
+
+from SIRF_torch import *
```

Comment:
let's not do that.

Reply:
sorry test cases is old I will remove that file.